extensions/azurePublishNew/src/node/azureResources/webApp.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  ProvisionMethod,
+  ProvisionWorkingSet,
+  ResourceConfig,
+  ResourceDefinition,
+  ResourceProvisionService,
+} from '../provisionService';
+import { AZURE_HOSTING_GROUP_NAME } from '../getResources';
+import { parseRuntimeKey } from '../../../../../Composer/packages/lib/shared';
+
+type WebAppConfig = ResourceConfig & {
+  key: 'webApp';
+  name: string;
+  location: string;
+  resourceGroupName: string;
+};
+
+export type WebAppResult = {
+  key: 'webApp';
+  endpoint: string;
+  hostname: string;
+};
+
+export const webAppResourceDefinition: ResourceDefinition = {
+  key: 'webApp',
+  description:
+    'App Service Web Apps lets you quickly build, deploy, and scale enterprise-grade web, mobile, and API apps running on any platform. Hosting for your bot.',
+  text: AZURE_HOSTING_GROUP_NAME,
```

Comment:
I don't think the text and the group name will be the same.  Even if they are the same string, the text shouldn't reference the group name constant so they can vary independently.

Reply:
good point. this is how it was but I've updated it so that `text` is not tied to that constant anymore